clang/lib/Headers/stdbool.h

```diff
@@ -23,7 +23,7 @@
 #elif defined(__GNUC__) && !defined(__STRICT_ANSI__)
 /* Define _Bool as a GNU extension. */
 #define _Bool bool
-#if __cplusplus < 201103L
+#if defined(__cplusplus) && __cplusplus < 201103L
```

Comment:
Shouldn't this be `!defined(__cplusplus) || __cplusplus < 201103L`?

Reply:
See https://reviews.llvm.org/D131213#3702137